modules/system/ServiceProvider.php

```diff
@@ -328,9 +328,9 @@ protected function registerMailer()
         /*
          * Override standard Mailer content with template
          */
-        Event::listen('mailer.beforeAddContent', function ($mailer, $message, $view, $data, $raw) {
+        Event::listen('mailer.beforeAddContent', function ($mailer, $message, $view, $data, $raw, $plain) {
             $method = $raw === null ? 'addContentToMailer' : 'addRawContentToMailer';
-            return !MailManager::instance()->$method($message, $raw ?: $view, $data);
+            return !MailManager::instance()->$method($message, $raw ?: $view ?: $plain, $data, $view === null);
```

Comment:
Can you add a comment in here to explain `$view === null`'s purpose?

Reply:
Sure @LukeTowers, pushed a new commit.